account_multicurrency_revaluation/tests/test_currency_revaluation.py

```diff
@@ -0,0 +1,1009 @@
+# Copyright 2012-2018 Camptocamp SA
+# Copyright 2020 CorporateHub (https://corporatehub.eu)
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import exceptions, fields
+from odoo.tests import common
+
+
+class TestCurrencyRevaluation(common.TransactionCase):
```

Comment:
```suggestion
from odoo.tests import tagged

@tagged('post_install', '-at_install')
class TestCurrencyRevaluation(common.TransactionCase):
```

I have a error when I run test, please add tagged
![image](https://github.com/user-attachments/assets/730f7f02-c642-4aff-b89c-af87dc33dc60)

Reply:
please use CE version `nacha_entry_class_code` belong to EE modules